src/Crypto/Spake2/Groups/Ed25519.hs

```diff
@@ -340,7 +340,11 @@ That final check is a Python assertion,
 which would crash the program if incorrect.
 For programming convenience, I just skip these values.
 
-jml doesn't know what is meant by the 'order' of a point.
+The 'order' of a point \(x\( is the number \(n\) such that:
```

Comment:
`\(` and `\)` are Haddock markup to indicate start and end (respectively) of inline LaTeX markup. The trailing `\(` is wrong.